src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -811,6 +811,19 @@
   (deps/index-upstream-foreign-libs))
 
 ;; --------------------
+;; Introspection
+
+(defn ^:export get-arglists
+  "Return the argument lists for the given symbol as string."
+  [s]
+  (when-let [var (some->> s repl-read-string first (resolve-var @env/*compiler*))]
+    (if-not (:macro var)
+      (:arglists var)
+      ;; filtering &env &form for macros
+      (map #(filterv (complement '#{&env &form}) %)
```

Comment:
After convo on Slack, [this](https://github.com/anmonteiro/lumo/blob/master/src/cljs/snapshot/lumo/repl.cljs#L616) solution will be implemented instead of this ugly hack.